README.md

```diff
@@ -112,6 +113,66 @@ pip install --no-deps -e .
+|   |-- cross.py.................# |-- APIs for computing cross products of 6D quantities.
+|   |-- inertia.py...............# |-- APIs for creating and manipulating 6D inertia matrices.
+|   |-- joint_model.py...........# |-- APIs defining the supported joint model and the corresponding transformations.
+|   |-- quaternion.py............# |-- APIs for creating and manipulating quaternions.
+|   |-- rotation.py..............# |-- APIs for creating and manipulating rotation matrices.
+|   |-- skew.py..................# |-- APIs for creating and manipulating skew-symmetric matrices.
+|   `-- transform.py.............# `-- APIs for creating and manipulating homogeneous transformations.
+|-- mujoco.......................# Package containing utilities to interact with the Mujoco passive viewer.
+|   |-- loaders.py...............# |-- Utilities for converting JaxSim models to Mujoco models.
+|   |-- model.py.................# |-- Class providing high-level methods to compute quantities using Mujoco.
+|   `-- visualizer.py............# `-- Class that simplifies opening the passive viewer and record videos.
+|-- parsers......................# Package containing utilities to parse model descriptions (SDF and URDF models).
+|   |-- descriptions/............# |-- Package containing the intermediate representation of a model description.
+|   |-- kinematic_graph.py.......# |-- Definition of the kinematic graph associated to a parsed model description.
+|   `-- rod/.....................# `-- Package to create the intermediate representation from model descriptions using ROD.
+|-- rbda.........................# Package containing the low-level rigid body dynamics algorithms.
+|   |-- aba.py...................# |-- The Articulated Body Algorithm.
+|   |-- collidable_points.py.....# |-- Kinematics of collidable points.
+|   |-- contacts/................# |-- Package containing the supported contact models.
+|   |-- crba.py..................# |-- The Composite Rigid Body Algorithm.
+|   |-- forward_kinematics.py....# |-- Forward kinematics of the model.
+|   |-- jacobian.py..............# |-- Full Jacobian and full Jacobian derivative.
+|   |-- rnea.py..................# |-- The Recursive Newton-Euler Algorithm.
+|   `-- utils.py.................# `-- Common utilities used in the current package.
+|-- terrain......................# Package containing resources to specify the terrain.
+|   `-- terrain.py...............# |-- Classes defining the supported terrains.
+|-- typing.py....................# Module containing type hints.
+`-- utils........................# Package of common utilities.
+    |-- jaxsim_dataclass.py......# |-- Utilities to operate on pytree dataclasses.
+    |-- tracing.py...............# |-- Utilities to use when JAX is tracing functions.
+    `-- wrappers.py..............# `-- Utilities to wrap objects for specific use-cases on pytree dataclass attributes.
+```
```

Comment:
That's really helpful! What about rendering it in Markdown and leave just the command to extract the tree? It would yield something like:

<details><summary>MD Rendered</summary>
<p>

```sh
tree -L 2 -I "__pycache__" -I "__init__*" -I "__main__*" src/jaxsim
```

- **api/** - Package containing the main functional APIs.
  - *com.py* - APIs for computing quantities related to the center of mass.
  - *common.py* - Common utilities used in the current package.
  - *contact.py* - APIs for computing quantities related to the collidable points.
  - *data.py* - Class storing the data of a simulated model.
  - *frame.py* - APIs for computing quantities related to additional frames.
  - *joint.py* - APIs for computing quantities related to the joints.
  - *kin_dyn_parameters.py* - Class storing kinematic and dynamic parameters of a model.
  - *link.py* - APIs for computing quantities related to the links.
  - *model.py* - Class defining a simulated model and APIs for computing related quantities.
  - *ode.py* - APIs for computing quantities related to the system dynamics.
  - *ode_data.py* - Set of classes to store the data of the system dynamics.
  - *references.py* - Helper class to create references (link forces and joint torques).
  - *exceptions.py* - Module containing functions to raise exceptions from JIT-compiled functions.
- **integrators/** - Package containing the integrators used to simulate the system dynamics.
  - *common.py* - Common utilities used in the current package.
  - *fixed_step.py* - Fixed-step integrators (explicit Runge-Kutta schemes).
  - *variable_step.py* - Variable-step integrators (embedded Runge-Kutta schemes).
  - *logging.py* - Module containing logging utilities.
- **math/** - Package containing mathematical utilities.
  - *adjoint.py* - APIs for creating and manipulating 6D transformations.
  - *cross.py* - APIs for computing cross products of 6D quantities.
  - *inertia.py* - APIs for creating and manipulating 6D inertia matrices.
  - *joint_model.py* - APIs defining the supported joint model and the corresponding transformations.
  - *quaternion.py* - APIs for creating and manipulating quaternions.
  - *rotation.py* - APIs for creating and manipulating rotation matrices.
  - *skew.py* - APIs for creating and manipulating skew-symmetric matrices.
  - *transform.py* - APIs for creating and manipulating homogeneous transformations.
- **mujoco/** - Package containing utilities to interact with the Mujoco passive viewer.
  - *loaders.py* - Utilities for converting JaxSim models to Mujoco models.
  - *model.py* - Class providing high-level methods to compute quantities using Mujoco.
  - *visualizer.py* - Class that simplifies opening the passive viewer and record videos.
- **parsers/** - Package containing utilities to parse model descriptions (SDF and URDF models).
  - **descriptions/** - Package containing the intermediate representation of a model description.
  - *kinematic_graph.py* - Definition of the kinematic graph associated to a parsed model description.
  - **rod/** - Package to create the intermediate representation from model descriptions using ROD.
- **rbda** - Package containing the low-level rigid body dynamics algorithms.
  - *aba.py* - The Articulated Body Algorithm.
  - *collidable_points.py* - Kinematics of collidable points.
  - **contacts/** - Package containing the supported contact models.
  - *crba.py* - The Composite Rigid Body Algorithm.
  - *forward_kinematics.py* - Forward kinematics of the model.
  - *jacobian.py* - Full Jacobian and full Jacobian derivative.
  - *rnea.py* - The Recursive Newton-Euler Algorithm.
  - *utils.py* - Common utilities used in the current package.
- **terrain/** - Package containing resources to specify the terrain.
  - *terrain.py* - Classes defining the supported terrains.
- *typing.py* - Module containing type hints.
- **utils/** - Package of common utilities.
  - *jaxsim_dataclass.py* - Utilities to operate on pytree dataclasses.
  - *tracing.py* - Utilities to use when JAX is tracing functions.
  - *wrappers.py* - Utilities to wrap objects for specific use-cases on pytree dataclass attributes.

</p>
</details> 

~~~suggestion
```
- **api/** - Package containing the main functional APIs.
  - *com.py* - APIs for computing quantities related to the center of mass.
  - *common.py* - Common utilities used in the current package.
  - *contact.py* - APIs for computing quantities related to the collidable points.
  - *data.py* - Class storing the data of a simulated model.
  - *frame.py* - APIs for computing quantities related to additional frames.
  - *joint.py* - APIs for computing quantities related to the joints.
  - *kin_dyn_parameters.py* - Class storing kinematic and dynamic parameters of a model.
  - *link.py* - APIs for computing quantities related to the links.
  - *model.py* - Class defining a simulated model and APIs for computing related quantities.
  - *ode.py* - APIs for computing quantities related to the system dynamics.
  - *ode_data.py* - Set of classes to store the data of the system dynamics.
  - *references.py* - Helper class to create references (link forces and joint torques).
  - *exceptions.py* - Module containing functions to raise exceptions from JIT-compiled functions.
- **integrators/** - Package containing the integrators used to simulate the system dynamics.
  - *common.py* - Common utilities used in the current package.
  - *fixed_step.py* - Fixed-step integrators (explicit Runge-Kutta schemes).
  - *variable_step.py* - Variable-step integrators (embedded Runge-Kutta schemes).
  - *logging.py* - Module containing logging utilities.
- **math/** - Package containing mathematical utilities.
  - *adjoint.py* - APIs for creating and manipulating 6D transformations.
  - *cross.py* - APIs for computing cross products of 6D quantities.
  - *inertia.py* - APIs for creating and manipulating 6D inertia matrices.
  - *joint_model.py* - APIs defining the supported joint model and the corresponding transformations.
  - *quaternion.py* - APIs for creating and manipulating quaternions.
  - *rotation.py* - APIs for creating and manipulating rotation matrices.
  - *skew.py* - APIs for creating and manipulating skew-symmetric matrices.
  - *transform.py* - APIs for creating and manipulating homogeneous transformations.
- **mujoco/** - Package containing utilities to interact with the Mujoco passive viewer.
  - *loaders.py* - Utilities for converting JaxSim models to Mujoco models.
  - *model.py* - Class providing high-level methods to compute quantities using Mujoco.
  - *visualizer.py* - Class that simplifies opening the passive viewer and record videos.
- **parsers/** - Package containing utilities to parse model descriptions (SDF and URDF models).
  - **descriptions/** - Package containing the intermediate representation of a model description.
  - *kinematic_graph.py* - Definition of the kinematic graph associated to a parsed model description.
  - **rod/** - Package to create the intermediate representation from model descriptions using ROD.
- **rbda** - Package containing the low-level rigid body dynamics algorithms.
  - *aba.py* - The Articulated Body Algorithm.
  - *collidable_points.py* - Kinematics of collidable points.
  - **contacts/** - Package containing the supported contact models.
  - *crba.py* - The Composite Rigid Body Algorithm.
  - *forward_kinematics.py* - Forward kinematics of the model.
  - *jacobian.py* - Full Jacobian and full Jacobian derivative.
  - *rnea.py* - The Recursive Newton-Euler Algorithm.
  - *utils.py* - Common utilities used in the current package.
- **terrain/** - Package containing resources to specify the terrain.
  - *terrain.py* - Classes defining the supported terrains.
- *typing.py* - Module containing type hints.
- **utils/** - Package of common utilities.
  - *jaxsim_dataclass.py* - Utilities to operate on pytree dataclasses.
  - *tracing.py* - Utilities to use when JAX is tracing functions.
  - *wrappers.py* - Utilities to wrap objects for specific use-cases on pytree dataclass attributes.
~~~

Reply:
I didn't like the list of lists, gave it a try with a table in https://github.com/ami-iit/jaxsim/pull/193/commits/f3c16e8588adc0c02495829e00652ea70d612c41 with the help of chat gpt. What do you think? I'm not sure if I still prefer the verbatim version over the table.

Reply:
Yes, it's not bad, but probably the verbatim version is better. Maybe we can put it in a `<details>` tag?

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/193/commits/beaa2a310258a06ab7c15e96d7538b8102934fe4, let me know if this is ok and I'll squash the last commits together before merging.

Reply:
Yes, it looks better. Thanks a lot! Good to go for me